src/services/utilities.ts

```diff
@@ -1913,7 +1913,9 @@ namespace ts {
             for (const newImport of sortedNewImports) {
                 const insertionIndex = OrganizeImports.getImportDeclarationInsertionIndex(existingImportStatements, newImport);
                 if (insertionIndex === 0) {
-                    changes.insertNodeBefore(sourceFile, existingImportStatements[0], newImport, /*blankLineBetween*/ false);
+                    changes.insertNodeBefore(sourceFile, existingImportStatements[0], newImport, /*blankLineBetween*/ false, {

```

Comment:
Makes sense, but I think this need to be predicated upon `existingImportStatements[0] === sourceFile.statements[0]`. Counterexample test:

```ts
// Actual header comment, copyright myself 2020

const IM_THE_FIRST_STATEMENT = 0;

// Lodash is the best!
import { add } from 'lodash';
// Even better is when we use underscore too
import { subtract } from 'underscore';

autoImportMe();
```

Reply:
good catch.